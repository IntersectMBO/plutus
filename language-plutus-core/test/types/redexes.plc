(program 1.0.0
  (lam
    ds_3
    [[(lam a_0 (type) (lam b_1 (type) (all p_matchOut_2 (type) (fun (fun a_0 (fun b_1 p_matchOut_2)) p_matchOut_2)))) [(con integer) (con 64)]] [(con integer) (con 64)]]
    [
      [
        { ds_3 (fun (all a_4 (type) (fun a_4 a_4)) [(con integer) (con 64)]) }
        (lam
          a_5
          [(con integer) (con 64)]
          (lam
            b_6
            [(con integer) (con 64)]
            (lam thunk_7 (all a_8 (type) (fun a_8 a_8)) a_5)
          )
        )
      ]
      (abs a_9 (type) (lam x_10 a_9 x_10))
    ]
  )
)
