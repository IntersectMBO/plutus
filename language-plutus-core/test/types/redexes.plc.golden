(fun [[(lam a_0 (type) (lam b_1 (type) (all p_matchOut_2 (type) (fun (fun a_0 (fun b_1 p_matchOut_2)) p_matchOut_2)))) [(con integer) (con 64)]] [(con integer) (con 64)]] [(con integer) (con 64)])